### ISSUE TITLE
Short/quick swipe down in picture viewer just removes the background and does not close the activity

### ISSUE THREAD

**Issue by jahirfiquitiva** [dodgingpixels/Optic#30]
This is how it ends up looking like:
![screenshot_20170726-203459](https://user-images.githubusercontent.com/10360816/28653002-c50460de-7250-11e7-9b03-da4bccd25087.png)

**Comment by staber** [comment]
Yes, there's a set distance you need to move away from center for the activity to exit. This image just "settled" in the wrong position. Not sure how much this can be improved though. I'll look more into it.

**Comment by jahirfiquitiva** [comment]
@staber Great. Thanks
